### PR TITLE
Add UI for email proxy settings

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by hishamco** [comment]
Screenshot 2022-02-27 194737

**Comment by hishamco** [comment]
We could add a toggle checkbox to display proxy settings, but I find it's useful to put them underneath the server host / port

File: src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml [hunk 1]
```diff
@@ -49,27 +52,48 @@
...
+                                <span class="hint">@T["The proxy port is optional."]</span>
                             </div>
                         </div>
                     </div>
                 </div>
+
+                <div id="@Html.Id("SelectEncryptionMethod")" class="form-group" asp-validation-class-for="EncryptionMethod">
+                    <label asp-for="EncryptionMethod">@T["Encryption method to use"]</label>
+                    <select asp-for="EncryptionMethod" class="form-control" disabled="@Model.AutoSelectEncryption">
+                        <option value="@SmtpEncryptionMethod.None" selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod ? true : false)">
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml:83]: I think razor is clever enough to just use `selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod)"`

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml:83]: Saw that it was not part of the PR

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml:83]: Seb can we merge this, then I can look for the places that may need to change

**Comment by agriffard** [comment]
Crowdin Translations updated to include those added by this PR.

**Pull request merged**